### PR TITLE
feat(gatsby-plugin-netlify-cms): external module for netlify-cms

### PR DESCRIPTION
**Pull request by ekoeryanto**

### CODE REVIEW

File: packages/gatsby-plugin-netlify-cms/README.md [hunk 1]
```diff
@@ -187,6 +187,39 @@ plugins: [
 ]
 ```
 
+### `externals`
+
+(_optional_, default: `undefined`)
+
+Configure the external modules for the cms, if the value is an array it will be passed to [html-webpack-externals-plugin](https://npm.im/html-webpack-externals-plugin). `externals` object options, if the value is thruty it will externalize `netlify-cms` and `netlify-identity-widget` packages.
```
Comment (pieh) [inline packages/gatsby-plugin-netlify-cms/README.md]: typo in "thruty" :)

**Comment by pieh** [comment]
If I understand webpack `externals` correctly - people would need to add script tags for externals themselves? If so this probably would need to be captured in README as well?

**Comment by pieh** [comment]
Ah, scratch my previous comment - this is handled by https://www.npmjs.com/package/html-webpack-externals-plugin

**Comment by erquhart** [comment]
@ekoeryanto we should discuss this before implementing any API changes. Would you mind opening an issue to discuss on the Netlify CMS repo? You can also reach out on [Gitter](https://gitter.im/netlify/netlifycms).

**Pull request closed without merging**